config/Dockerfiles/rpmbuild/rpmbuild-test.sh

```diff
@@ -101,6 +101,8 @@ cp ${CURRENTDIR}/${fed_repo}/results_${fed_repo}/${VERSION}/*/*.rpm ${RPMDIR}/
 # Run rpmlint
 rpmlint ${RPMDIR}/ > ${LOGDIR}/rpmlint_out.txt
 pushd ${RPMDIR} && createrepo .
+mkdir logs
```

Comment:
do we not already have a logs directory?

Reply:
We do, but not in ${RPMDIR}. RPMDIR is the dir that gets rsynced to artifacts.ci, so right now it is just .rpms. This logs dir is not for archiving with the job run, but to get the logs up to the artifacts server so they are right next to the rpms. As far as I could tell, that is what we want. @arilivigni is that correct? Or was the goal just to have them in the jenkins build artifacts. I suppose I could put them there too...

Reply:
Yes that is exactly right so when we rsync for an rpm or also a tree the build logs of what occurred is already there.